src/firebase/api/gcp-utils.js

```diff
@@ -89,30 +88,18 @@ export const observeStimuliCompletion =
  */
 const getFileUrlsFromBucket = async (userId, experimentId) => {
```

Comment:
let's have a generic function getFileUrlsFromBucket where `userId` and `experimentId` are required and maybe a filter as optional? So we can use this to filter only images or something, then we can have another method called `getSieStimuliFromBucket`.

Reply:
sounds good to me, will update!